src/bot/commands/roulette.js

```diff
@@ -6,7 +6,7 @@ class Barrel {
   constructor(size = 6) {
     this.size = size
     this.slot = new Array(size).fill(false)
-    this.slot[Math.random() * this.size] = true
+    this.slot[Math.round(Math.random() * this.size)] = true
```

Comment:
Should we just use the `addBullet` method since we always shuffle before the shot?

Reply:
We can also add a bullet at index 0. btw the shuffle will do his job. No reason to place at random index.

Another approach is to add a second parameter to add a predefined number of bullet at start